core/lib/zksync_core/src/sync_layer/metrics.rs

```diff
@@ -8,7 +8,7 @@ use zksync_types::aggregated_operations::AggregatedActionType;
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, EncodeLabelValue, EncodeLabelSet)]
 #[metrics(label = "stage", rename_all = "snake_case")]
 pub(super) enum FetchStage {
-    GetMiniblockRange,
+    GetL2BlockRange,
```

Comment:
I'm not sure if it breaks any dashboard, maybe makes sense to keep old metric name